### PR TITLE
Styling contact us page

### PR DESCRIPTION
**Pull request by Yehezkiel01**

### CODE REVIEW

File: src/routes/contact.svelte [hunk 1]
```diff
@@ -1,18 +1,41 @@
+<style>
+    .social {
+        display: flex;
+        justify-content: center;
+        margin: 3rem 10rem;
+    }
+
+    .social a {
+        margin: 0 auto;
```
Comment (benclmnt) [inline src/routes/contact.svelte]:
```suggestion
        padding: 1rem;
```

File: src/routes/contact.svelte [hunk 2]
```diff
@@ -1,18 +1,41 @@
+<style>
+    .social {
+        display: flex;
+        justify-content: center;
+        margin: 3rem 10rem;
```
Comment (benclmnt) [inline src/routes/contact.svelte]:
```suggestion
```

**Review by benclmnt — CHANGES_REQUESTED** [review]
Consider adding a breakpoint scenario for smaller size screens

Proposed solutions:
1. Change the orientation of the flex box orientation on mobile
2. Rather than inlining the width for each image, maybe you can add it as a class property, that can easily be changed for different media width using media query.

**Comment by Yehezkiel01** [comment]
> Consider adding a breakpoint scenario for smaller size screens
> 
> Proposed solutions:
> 
> 1. Change the orientation of the flex box orientation on mobile
> 2. Rather than inlining the width for each image, maybe you can add it as a class property, that can easily be changed for different media width using media query.

Thx for the suggestion. I was thinking of a simpler solution by using `flex-flow: row wrap;` to make them wrap when the screen gets smaller for phone.

What do you think about this? [I have just pushed the wrap version]

**Review by benclmnt — APPROVED** [review]
Looks good!

**Pull request merged**